revpi_hat/hat.py

```diff
@@ -0,0 +1,122 @@
+from datetime import date
+import json
+from typing import List
+
+
+class RevPiHatEEPROMAttributeException(Exception):
+    pass
+
+
+class RevPiHatEEPROM:
+    def __init__(self, hat_eeprom_path: str = "/proc/device-tree/hat/", load_contents: bool = True) -> None:
```

Comment:
the second parameter "load_contents" might be used more often. Would be better to put it ahead to benefit the case to specify only not load the profs.  I guess when the hat_eeprom_path get changed by raspberrypi fundation, we would change the default value here.

Reply:
I can agree. Will swap the arguments